src/classes/PREN_Affiliation_TEST.cls

```diff
@@ -61,12 +61,12 @@ public with sharing class PREN_Affiliation_TEST {
         Test.stopTest();
         
         //An Affiliation should have been automatically created fron the Program Enrollment
-        List<Affiliation__c> affls = [SELECT Contact__c, Account__c FROM Affiliation__c WHERE Account__r.ID = :acc.ID];
+        List<Affiliation__c> affls = [SELECT Contact__c, Account__c FROM Affiliation__c WHERE Account__c = :acc.ID];
```

Comment:
Even though __r.ID is technically correct, for some reason it just really bothers me so I changed it 😬

Reply:
I'm fine with that and prefer it.